config/ember-try.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
```

Comment:
@mansona Totally baffled why these got missed...thanks for catching it and posting them for me!!!